### PR TITLE
[DOCS-4903] Add aliases and links to Getting Started with Obs Pipelines  

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

- Adds previous aliases in Integrate Observability Pipelines Worker and Datadog to the new Getting Started with Obs Pipelines.
- Add links to Getting Started with Obs Pipelines in further reading sections.

Ready to merge.

### Motivation

[Docs-4903](https://datadoghq.atlassian.net/browse/DOCS-4903)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/observability_pipelines/working_with_data.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ kind: Documentation
 aliases:
   - /integrations/observability_pipelines/working_with_data/
 further_reading:
+  - link: /observability_pipelines/installation/
```
Comment (drichards-87) [inline content/en/observability_pipelines/working_with_data.md:7]: Quick question... Did you mean to link this to the Installation doc or the Getting Started guide?

File: content/en/agent/_index.md [hunk 2]
```diff
@@ -64,7 +64,8 @@ Datadog Agent release numbering follows <a href="https://semver.org/">SemVer</a>
   {{< nextlink href="/agent/versions/">}}<u>Versions</u>: Agent 7 is the latest major version of the Datadog Agent. Learn about changes between major Agent versions and how to upgrade.{{< /nextlink >}}
   {{< nextlink href="/agent/troubleshooting">}}<u>Troubleshooting</u>: Find troubleshooting information for the Datadog Agent.{{< /nextlink >}}
   {{< nextlink href="/agent/guide">}}<u>Guides</u>: These are in-depth, step-by-step tutorials for using the Agent.{{< /nextlink >}}
-  {{< nextlink href="/agent/security">}}<u>Security</u>: Information on the main security capabilities and features available to customers to ensure their environment is secure.{{< /nextlink >}}
+  {{< nextlink href="/agent/security">}}<u></u>: Information on the main security capabilities and features available to customers to ensure their environment is secure.{{< /nextlink >}}
```
Comment (drichards-87) [inline content/en/agent/_index.md:67]: Looks like we might be missing the title for this link.

**Review by drichards-87 — APPROVED** [review]
Left you two small pieces of feedback.

File: content/en/agent/_index.md (hunk 2 above)
Comment (maycmlee) [reply content/en/agent/_index.md:67]: 🤦‍♀️ Added "Security" back in.

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**